clustered/integration-test/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
   testCompile "com.fasterxml.jackson.core:jackson-databind:2.8.0"
   testRuntime project(':clustered:clustered-dist')
   testRuntime project(':dist')
+  testRuntime "org.terracotta:runnel:$terracottaPlatformVersion"
```

Comment:
How come this is the only change needed and not around the scope of the `compileOnly` ones?

Reply:
I don't know. I just know that adding only this worked. Unless IntelliJ remembered a bit too much the state with all `compileOnly` removed. Try it for yourself.

Reply:
Ok did a test, it works if that is declared as `testCompileOnly` which allows it to not leak in the normal gradle build - so please change it.

Reply:
Now in `testCompileOnly`